tests/ui/inline-const/elided-lifetime-being-infer-vars.rs

```diff
@@ -0,0 +1,11 @@
+// run-pass
```

Comment:
this doesn't need to be run-pass

```suggestion
// check-pass
```

Reply:
Fixed